src/pages/patientView/clinicalInformation/ClinicalInformationSamplesTable.tsx

```diff
@@ -12,48 +13,66 @@ type TODO = any;
 
 export class ClinicalInformationSamplesTable extends React.Component<IClinicalInformationSamplesTableProps, any> {
 
-    render() {
+    public render() {
 
-        const sampleTableData = convertSamplesData(this.props.samples);
+        const sampleInvertedData: TODO = convertSamplesData(this.props.samples);
```

Comment:
This is admittedly kind of weird.  How we seem to prepare the data twice.  I'll take a look again after merge of @adufilie 's ts branch, in which I know this code was touched